### PR TITLE
Add Avatar and Executives page component

### LINKED ISSUE #19: Create About Us Page Executives Section
image

### PR DESCRIPTION
**Pull request by tylerlam1**
Resolves #19.

**Web Page**
Screen Shot 2022-08-08 at 11 37 49 AM

**Mobile Page**
Top:
Screen Shot 2022-08-08 at 2 21 21 PM

Bottom:
Screen Shot 2022-08-08 at 2 21 26 PM

Double checked to make sure links work as well.

### CODE REVIEW

File: pages/_app.js [hunk 1]
```diff
@@ -13,9 +13,6 @@ function MyApp({ Component, pageProps }) {
       <Navbar />
       <Component {...pageProps} />
       {/* CONTAINER FOR CONTENT BELOW TO TEST NAVBAR IS IN RIGHT POSITION*/}
```
Comment (harsohailB) [inline pages/_app.js]: Can we also get rid of this?

File: components/Executives.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import Heading from "./Heading";
+import Avatar from "./Avatar";
+import { EXECUTIVES } from "../data/executives";
+
+const EXECUTIVES_CONTAINER = "bg-white flex flex-col items-center mb";
+const EXECUTIVE_LEVEL_CONTAINER = "flex flex-row flex-wrap justify-center w-full sm:w-8/12 mb-5";
+
+const Executives = () => {
```
Comment (harsohailB) [inline components/Executives.jsx:8]: Don't see this rendered anywhere. Pull in `main` and place in `pages/who-we-are/index.js`

**Review by harsohailB — CHANGES_REQUESTED** [review]

File: components/Executives.jsx (hunk 2 above)
Comment (tylerlam1) [reply components/Executives.jsx:8]: Will do

Comment (tylerlam1) [reply components/Executives.jsx:8]: Done

File: components/Avatar.jsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import Link from "next/link";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faLinkedin } from "@fortawesome/free-brands-svg-icons";
+
+const PROFILE_CONTAINER = "flex flex-col items-center m-10";
+
+const Avatar = ({ name, file, position, linkedin }) => {
+  const FontAwesomeLink = ({ username, icon }) => (
+    <Link href={`https://linkedin.com/in/${username}`}>
```
Comment (harsohailB) [inline components/Avatar.jsx]: nit: open in new tab?

**Review by harsohailB — APPROVED** [review]
Amazing! Good work! 🚢

File: components/Avatar.jsx (hunk 3 above)
Comment (topanb11) [reply components/Avatar.jsx]: agreeing with this

**Review by topanb11 — APPROVED** [review]
I can't see the execs in most recent preview but previous one looks fire

File: components/Avatar.jsx (hunk 3 above)
Comment (tylerlam1) [reply components/Avatar.jsx]: Sounds g

**Pull request merged**